enhancements/network/network-policy-audit-logging.md

```diff
@@ -0,0 +1,315 @@
+---
+title: audit-logging-of-network-policy-events
+authors:
+  - "@astoycos"
+reviewers:
+  - "@abhat" 
+  - "@vpickard"
+  - "@trozet" 
+  - "@Billy99"
+approvers:
+  - "@knobunc"
+creation-date: 2020-12-11
+last-updated: 2020-12-11
+status: implementable
+
+---
+
+# Audit Loggging of Network Policy Events 
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
```

Comment:
You have it marked implementable above.  In general, update this checklist as appropriate